test/modules/sprint.spec.js

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+import sprint from '@/modules/sprint'
+import plateBarcode from '@/modules/plate_barcode'
+import config from '@/nuxt.config'
+
+jest.mock('@/modules/plate_barcode')
+
+// TODO: move out into helper
+const errorResponse = new Error('There was an error')
+const rejectPromise = () => Promise.reject(errorResponse)
+
+const layout = {
+  barcodeFields: [
+    {
+      x: 16,
+      y: 1,
+      cellWidth: 0.2,
+      barcodeType: 'code39',
+      value: 'DN111111',
+      height: 5
+    }
+  ],
+  textFields: [
+    {
+      x: 3,
+      y: 3,
+      value: 'DN111111',
+      font: 'proportional',
+      fontSize: 1.7
+    },
+    {
+      x: 57,
+      y: 3,
+      value: 'LHTR',
+      font: 'proportional',
+      fontSize: 1.7
+    }
+  ]
+}
+
+describe('Sprint', () => {
+  it('#createLayout', () => {
+    expect(sprint.createLayout('DN111111', 'DN222222')).toEqual(layout)
+  })
+
+  describe('#createBarcodes', () => {
+    it('will produce a single barcode with no arguments', () => {
+      expect(sprint.createBarcodes()).toEqual(['DN111111'])
```

Comment:
`createBarcodes` with no arguments defaults to one barcode. I know this is just a place holder method, so not sure what the indended use is - but if it is relevant/ the behaviour for the plate_barcode module method is the same, might be worth adding a comment

Reply:
I think I will remove it. I used it to create barcodes before I added the plate barcode module.